lib/stitches/spec/test_headers.rb

```diff
@@ -34,6 +34,8 @@ def mime_type(options)
   end
 
   def set_authorization_header(headers,options)
+    return nil if Stitches.configuration.disable_api_key_support
```

Comment:
This is probably my least favorite portion of this. It's not strictly necessary, but having this set the authorization header to nil when api key support is disable seems nice.

Reply:
Let me know if I'm reading this wrong: would this make alternatively make sense?

```diff
}.tap { |headers|
-  set_authorization_header(headers,options)
+  set_authorization_header(headers,options) if Stitches.configuration.disable_api_key_support
}
```